### DISCUSSION TITLE
Handlesubmit not responding

### DISCUSSION THREAD

**Discussion by davidshare** [react-hook-form/react-hook-form#3735] [Q&A] [answered]
I have a form I manage with react-hook-form and ant design. Validation is done with Joi as the resolver. After filling the form and I hit the submit button, there is no response. It is almost as if the button was not hit. Even a console log in the handle submit method is not triggered. I need help to figure out what I am getting wrong.

Here is a codesandbox:

https://codesandbox.io/s/romantic-cherry-2y24u?file=/src/App.js

Thanks in advance.

**Comment by markivancho** [comment, marked as answer]
Probably something specific to ant library. But everything works fine if you move `handleSubmit` from Form's onSubmit prop to Button's `onClick`
![image](https://user-images.githubusercontent.com/30623974/102690310-310ee580-420d-11eb-919c-3307ce5d689a.png)

Edit: 
seems like ant Form doesn't have onSubmit prop - use onFinish instead
![image](https://user-images.githubusercontent.com/30623974/102690699-e2af1600-420f-11eb-9952-b9a47ff1537c.png)

with onFinish on CSB works same as first answer.

**Reply by davidshare** [reply]
Thank you.

**Reply by bluebill1049** [reply]
Thank you @markivancho
